db/seed/RbacRole.csv

```diff
@@ -1,5 +1,5 @@
 Id;OwnerId;CreatedAt;UpdatedAt;Name;Description;Type
-F67CACC9-8DD4-4481-AC15-00B5DD83B046;0000DFD5-0000-0000-0000-00000000000A;2000-01-01;2000-01-01;Guest;Default guest permissions;System
-22DAB91B-C2D8-4840-A173-1416EF1B882D;0000DFD5-0000-0000-0000-00000000000A;2000-01-01;2000-01-01;Reader;Default reader permissions;System
-2C561A6D-90F4-4649-80B3-76A854A64EA2;0000DFD5-0000-0000-0000-00000000000A;2000-01-01;2000-01-01;Contributor;Default contributor permissions;System
-36202DFB-D106-440D-8B99-F11BC8D77C9C;0000DFD5-0000-0000-0000-00000000000A;2000-01-01;2000-01-01;Owner;Default owner permissions;System
+36202DFB-D106-440D-8B99-F11BC8D77C9C;44302b63-b9fd-47d8-a6f9-cfe29213d92e;2025-08-21T07:40:23.155256;2025-08-21T07:40:23.155367;Owner;Full access to all resources;System
+2C561A6D-90F4-4649-80B3-76A854A64EA2;44302b63-b9fd-47d8-a6f9-cfe29213d92e;2025-08-21T07:40:23.155372;2025-08-21T07:40:23.155372;Contributer;Can modify existing resources;System
+22DAB91B-C2D8-4840-A173-1416EF1B882D;44302b63-b9fd-47d8-a6f9-cfe29213d92e;2025-08-21T07:40:23.155375;2025-08-21T07:40:23.155376;Reader;Read-only access;System
+F67CACC9-8DD4-4481-AC15-00B5DD83B046;44302b63-b9fd-47d8-a6f9-cfe29213d92e;2025-08-21T07:40:23.155379;2025-08-21T07:40:23.155379;Guest;Very limited access;System
```

Comment:
Any reason for the change in OwnerId? I intentionally made it somewhat easy to spot that it's a system role by making it '0000DFD5-0000-0000-0000-00000000000A'

Reply:
That is a fair point; Fixed!